src/tea_json.res

```diff
@@ -3,16 +3,16 @@ module Decoder = {
 
   module ObjectDict = Map.Make(String)
 
-  type t<'input, 'result> = Decoder('input => Tea_result.t<'result, error>)
+  type t<'input, 'result> = Decoder('input => Belt.Result.t<'result, error>)
```

Comment:
Are you able to use `result<'x,'y>` instead of `Belt.Result.t<'x,'y>`?

Reply:
Yes I have tried it and it works I will change it.